doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,41 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Load Balancer Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The table below specifies a set of requirements for the Load Balancer platform service.
+
+========== ===================================================================== ============================================================================
+Reference  Requirement                                                           Notes
+========== ===================================================================== ============================================================================
+pas.lb.001 The Load Balancer must support resource scaling	
+pas.lb.002 The Load Balancer must support resource resiliency	
+pas.lb.003 The Load Balancer must support scaling and resiliency in the local    Define local environment: within a subnet, tenant network, Availability 
+           environment                                                           Zone of a cloud, ...
+pas.lb.004 The Load Balancer should support resiliency in the global environment	
+pas.lb.005 The Load Balancer must support OSI Level 3/4 load-balancing and may   OSI Level 3 load-balancing decision on the source and destination IP 
+           support Level 7 Load Balancing                                        addresses and OSI Level 4 TCP port numbers. <br>OSI Level 7 (application
+                                                                                 characteristics based) should support HTTP and HTTPS
+pas.lb.006 The Load Balancer must, at a minimum, support round-robin load-
+           balancing	
+pas.lb.007 The Load Balancer must create event logs with the appropriate 
+           severity levels (catastrophic, critical, ...)	
+pas.lb.008 The Load Balancer must support monitoring of endpoints	
+pas.lb.009 The Load Balancer must support Direct Server Return (DSR)             Other modes OK as well, but DSR should always be supported
+pas.lb.010 The Load Balancer must state-full TSP load-balancing	
+pas.lb.011 The Load Balancer must support UDP load-balancing	
+pas.lb.012 The Load Balancer must support load-balancing and correct handling 
+           of fragmented packets 	
+pas.lb.013 The Load Balancer may support state-full SCTP load-balancing	
+pas.lb.014 The Load Balancer may support state-full M-TCP load-balancing	
+pas.lb.015 The L7 Load Balancer may support HTTP2	
+pas.lb.016 The L7 Load Balancer may support HTTP3	
+pas.lb.017 The L7 Load Balancer may support QUIC
+========== ===================================================================== ============================================================================
+
+**Table 5-7b:** Platform Services - Load Balancer Requirements.
```

Comment:
Are there any open source solutions implementing these requirements?

Reply:
@CsatariGergely Not every requirement but then these are not mandatory requirements. The Previous section lists the services and examples of open-source sw.

Reply:
@CsatariGergely, are you satisfied with the reposne form @pgoyal01 ?  If yes, please approve.  Otherwise, let us discuss what to do.

Reply:
I really think that Anuket should define requirements implementable using open source components. I do not think to be usefull to require something (even with should) in one document and define exceptions in an other one. 

By checking Octavia and NGINX as references in open source I think the following requirements are not supported in open source: 

- stateful TCP load-balancing (only NGINX supports it) 
- stateful M-TCP
- HTTP3
- QUIC

I suggest to remove these requirements.

Also Level 7 is too generic in pas.lb.015, the requirement should be about http and https